src/content/reference/fleet management/clusters/introduction_cluster_api.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Introduction to Cluster API
+description: A throughout explanation of how our platform implements Cluster API standard.
+weight: 100
+last_review_date: 2024-03-07
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-docs
+user_questions:
+  - How does Giant Swarm serve Cluster API features?
+  - What other services are needed in a platform to complete Cluster API?
+---
+
+The [Cluster API project](https://cluster-api.sigs.k8s.io/) is indeed a Kubernetes sub-project that presents an innovative approach to provisioning, configuring, and managing clusters. It employs a declarative model, which means that users can define the desired state of their clusters, and the system will automatically work to achieve that state. This powerful tool offers a standard mechanism to handle Kubernetes clusters, simplifying operations and increasing efficiency.
+
+Cluster API leverages the concept of operators and Custom Resource Definitions (CRDs) to manage Kubernetes clusters. An operator is essentially a controller that responds to cluster lifecycle events, allowing for automated creation, configuration, and management of clusters.
+
+Custom Resource Definitions (CRDs) are extensions of the Kubernetes API that allow the creation of new resources - in this case, the cluster resources. When a user defines a cluster configuration using these custom resources, the Cluster API operator reacts to this declarative statement and works to achieve the desired state of the cluster. This includes tasks such as provisioning infrastructure, bootstrapping nodes, and applying configurations.
+
+## Giant Swarm flavour
+
+In our platform we offer the Cluster API features with some additional sugar on top to complete some capabilities we consider as must-have for a complete production system. Next we list the features coming with our Cluster API implementation:
```

Comment:
```suggestion
The Giant Swarm platform offers additional capabilities on top of Cluster API that we consider as must-have for a complete production system.

We leverage these features of Cluster API:
```

Avoiding colloquial terms that could be interpreted differently (additional sugar is bad, syntactic sugar is good, ... 😆).